app/src/main/res/values/strings.xml

```diff
@@ -42,4 +42,24 @@
     <string name="save_lrc_file">Save .lrc file</string>
     <string name="no_results">No results</string>
     <string name="no_internet_server">No internet connection or server is down</string>
+    <string name="lyrics_not_found">Lyrics not found</string>
+    <string name="unknown">Unknown</string>
+    <string name="lyrics_saved_to">Lyrics saved to</string>
+    <string name="generated_using">Generated using SongSync</string>
```

Comment:
mark translatable="false", this should not be localized (may cause issues with utf8 langs like persian, chinese, etc), is inside file so should not be dependant on UI language

Reply:
Okay!